src/commands/runtime/action/create.js

```diff
@@ -213,6 +231,9 @@ ActionCreate.flags = {
     description: 'parameter values in KEY VALUE format', // help description for flag
     multiple: true // allow setting this flag multiple times
   }),
+  copy: flags.string({
+    description: 'treat ACTION as the name of an existing action' // help description for flag
```

Comment:
I think this can be clearer, as to what the flag actually does, simply. "copy an existing action"